### PR TITLE
chore: update theme

### PR DESCRIPTION
**Pull request by fzipi**
## Proposed changes

- updates theme
- update hugo version

## Further comments

- needed to fix some relative references and config changes

### CODE REVIEW

File: .github/workflows/gh-pages.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      HUGO_VERSION: 0.122.0
+      HUGO_VERSION: 0.135.1
```
Comment (fzipi) [inline .github/workflows/gh-pages.yml:35]:
```suggestion
      HUGO_VERSION: 0.135.0
```

File: .github/workflows/gh-pages.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      HUGO_VERSION: 0.122.0
+      HUGO_VERSION: 0.135.0
```
Comment (fzipi) [inline .github/workflows/gh-pages.yml:35]:
```suggestion
      # renovate: datasource=github-releases depName=gohugo/hugo
      HUGO_VERSION: 0.135.0
```

File: content/deployment/extended_install.md [hunk 3]
```diff
@@ -202,7 +202,7 @@ mv rules/RESPONSE-999-EXCLUSION-RULES-AFTER-CRS.conf.example rules/RESPONSE-999-
 
 The engine should support the `Include` directive out of the box. This directive tells the engine to parse *additional* files for directives. The question is where to put the CRS rules folder in order for it to be included.
 
-Looking at the CRS files, there are quite a few ".conf" files. While the names attempt to do a good job at describing what each file does, additional information is available in the [rules]({{< ref "../rules/" >}}) section.
+Looking at the CRS files, there are quite a few ".conf" files. While the names attempt to do a good job at describing what each file does, additional information is available in the [rules]({{% ref "rules" %}}) section.
```
Comment (theseion) [inline content/deployment/extended_install.md:205]: Does Hugo flatten the page tree suddenly? Or does it have "magic lookup"?

Comment (fzipi) [reply content/deployment/extended_install.md:205]: This looks like a legacy magic miracle, when docs where in the same place as the rules. Clearly this won't work. But we can fix this in a followup, let me create an issue.

Comment (fzipi) [reply content/deployment/extended_install.md:205]: Issue created.

**Review by theseion — APPROVED** [review]

**Pull request merged**